packages/broadway.rb

```diff
@@ -15,36 +15,32 @@ class Broadway < Package
   depends_on 'gtk3'
 
   def self.build
-    system "echo 'GDK_BACKEND=broadway' > .broadway.env"
-    system "echo 'XDG_RUNTIME_DIR=/var/run/chrome' >> .broadway.env"
-    system "echo 'BROADWAY_DISPLAY=:5' >> .broadway.env"
+    system "echo 'export GDK_BACKEND=broadway' > .broadway.env"
+    system "echo 'export XDG_RUNTIME_DIR=/var/run/chrome' >> .broadway.env"
+    system "echo 'export BROADWAY_DISPLAY=:5' >> .broadway.env"
```

Comment:
This is not how an environment file works.  Please spend some time and learn the correct way.

Reply:
Excuse me? I probably know more about environment variables than you do.

Reply:
Whatever.  The `export` is not needed.

Reply:
Yes, it is. Try using it without `export`. Shell variables are different than environment variables. Programs can only interact with environment variables.

Reply:
If you don't use `export`, the variables only stay in the current shell, not on to child processes. Say if I wrote two bash scripts like this:
`calling.sh`:
```
#!/bin/bash

CHILDREN="how ever many you want"
./how_many.sh
```
`how_many.sh`:
```
#!/bin/bash

if [[ -z "${CHILDREN}" ]]; then
  CHILDREN="none at all"
fi

echo "${CHILDREN}"
```
It would echo "none at all".
However, if I added `export ` at the beginning of line 3 of `calling.sh`, the output would be "how ever many you want".

Reply:
If we only set `DISPLAY` / any of the other variables when we're **calling** the daemon, it will take ***no effect*** on the programs with which we are using the daemon.

Reply:
I totally understand what you are talking about.  Take a look at the output of `ps ax|grep broadway`.  Do you see `broadwayd :5`?  If you do, then the environment variable is working as expected.

Reply:
I do, however GTK applications **do not** show up in the browser window. With this fix, they do.

Reply:
```
Loading extra module: /usr/share/crosh/dev.d/50-crosh.sh
Welcome to crosh, the Chrome OS developer shell.

If you got here by mistake, don't panic!  Just close this tab and carry on.

Type 'help' for a list of commands.

If you want to customize the look/behavior, you can use the options page.
Load it by using the Ctrl+Shift+P keyboard shortcut.

crosh> shell
broadwayd process 26399 is running
chronos@localhost / $ qemu-system-i386 
chronos@localhost / $ cat ~/.broadway.env 
GDK_BACKEND=broadway
XDG_RUNTIME_DIR=/var/run/chrome
BROADWAY_DISPLAY=:5
chronos@localhost / $ qemu-system-i386 
chronos@localhost / $ echo $GDK_BACKEND
broadway 
chronos@localhost / $ export GDK_BACKEND
chronos@localhost / $ qemu-system-i386 
Unable to init server: Could not connect: Connection refused
gtk initialization failed
chronos@localhost / $ export BROADWAY_DISPLAY
chronos@localhost / $ qemu-system-i386 

** (qemu-system-i386:26736): WARNING **: 11:11:37.736: experimental: using broadway, x11 virtual keysym
mapping - with very limited support. See also
https://bugzilla.gnome.org/show_bug.cgi?id=700105
```
When there's no error from `qemu-system-i386`, it runs in the display, not the browser window.

Reply:
Fair enough.  I just tested and you are correct about gtk apps in the browser.  See #2950.